extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/QuarkusCurrentRequest.java

```diff
@@ -22,8 +24,12 @@ public ResteasyReactiveRequestContext get() {
     @Override
     public void set(ResteasyReactiveRequestContext set) {
         if (set == null) {
-            currentVertxRequest.setOtherHttpContextObject(null);
-            currentVertxRequest.setCurrent(null);
+            try {
+                currentVertxRequest.setOtherHttpContextObject(null);
+                currentVertxRequest.setCurrent(null);
+            } catch (ContextNotActiveException ignored) {
+
```

Comment:
Maybe a comment explaining why we ignore it?

Reply:
Done